custom-connectors/CardPlatform/readme.md

```diff
@@ -0,0 +1,74 @@
+## CardPlatform Adaptive Card Connector
```

Comment:
@st0014 The image URL in the readme.md are broken, please fix them.

Reply:
Updated the readme.md file with the correct path to the images